### PR TITLE
PMM-4702: Update mongo-driver to 1.1.1

### PR DESCRIPTION
**Pull request by DexterHD**

### CODE REVIEW

**Comment by AlekSi** [comment]
https://github.com/Percona-Lab/pmm-submodules/pull/472. If it works, we will release 0.10 tomorrow morning

**Comment by DexterHD** [comment]
> So, we will provide tls options via DSN, right?

I'm not sure how it should be. The "right way" from `mongodb` perspective is to use `connString`. Maybe we need to duplicate flags or map it somehow, but I don't know.

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+## [0.10.0]
+### Changed
+- `go.mongodb.org/mongo-driver` was updated to `v1.1.1`.
+- All `--mongodb.tls*` flags was removed. Use [tls-options](https://docs.mongodb.com/manual/reference/connection-string/#tls-options) instead.
```
Comment (AlekSi) [inline CHANGELOG.md]: were

File: collector/mongodb_collector.go [hunk 2]
```diff
@@ -53,16 +48,11 @@ type MongodbCollectorOpts struct {
...
-		TLSPrivateKeyFile:     in.TLSPrivateKeyFile,
-		TLSCaFile:             in.TLSCaFile,
-		TLSHostnameValidation: in.TLSHostnameValidation,
-		PoolLimit:             in.DBPoolLimit,
-		SocketTimeout:         in.SocketTimeout,
-		SyncTimeout:           in.SyncTimeout,
-		AuthentificationDB:    in.AuthentificationDB,
+		URI:                in.URI,
+		PoolLimit:          in.DBPoolLimit,
+		SocketTimeout:      in.SocketTimeout,
```
Comment (AlekSi) [inline collector/mongodb_collector.go]: Those things are parts of URI too. If drivers supports then, we should remove them in 0.11 (later).

**Comment by AlekSi** [comment]
Please make a release and a new branch for PMM ASAP.

File: collector/mongodb_collector.go (hunk 2 above)
Comment (DexterHD) [reply collector/mongodb_collector.go]: Ok. I've made a ticket.

**Pull request merged**